src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package qna.domain;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static qna.domain.AnswerTest.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+class AnswerRepositoryTest {
+    @Autowired
+    AnswerRepository answerRepository;
+
+    @Test
+    void save_and_find() {
+        Answer actual = answerRepository.save(A1);
+        assertAll(
+            () -> assertThat(actual.getId()).isNotNull(),
+            () -> assertThat(actual.getCreatedAt()).isNotNull(),
+            () -> assertThat(actual.getUpdatedAt()).isNotNull(),
+            () -> assertThat(actual.getUpdatedAt()).isEqualTo(actual.getCreatedAt()),
+            () -> assertThat(actual.isDeleted()).isFalse(),
+            () -> assertThat(actual.getQuestionId()).isEqualTo(A1.getQuestionId()),
+            () -> assertThat(actual.getWriterId()).isEqualTo(A1.getWriterId())
+        );
+        assertThat(answerRepository.findByIdAndDeletedFalse(actual.getId()).orElseThrow(RuntimeException::new))
+            .isEqualTo(actual);
```

Comment:
assertj 에서는 `Optional` 에 대한 기능도 제공해주고 있어서 다음과 같이 검증도 가능합니다!
참고만해주세요!!
```suggestion
        assertThat(answerRepository.findByIdAndDeletedFalse(actual.getId()))
                .contains(actual);
```